frontend/src/homepage/DevicesCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Grid } from "@material-ui/core";
+
+import DashboardCard from "../general/dashboard/DashboardCard";
+import DashBoardButton from "../general/dashboard/DashboardButton";
+
+export default function DevicesCard() {
+  return (
+    <DashboardCard title="Devices">
+      <Grid container justify="center" direction="row" spacing={3}>
+        <Grid item xs={4}>
+          <DashBoardButton href="/Devices">View Senders</DashBoardButton>
+        </Grid>
+        <Grid item xs={4}>
+          <DashBoardButton href="/Devices">View Receivers</DashBoardButton>
```

Comment:
do we want to make a bug for skipping to the receivers tab on the devices page properly?

Reply:
Created issue #359